benchmark/sample.sh

```diff
@@ -0,0 +1,29 @@
+# ClinVar
+wget -qO- 'https://ftp.ncbi.nlm.nih.gov/pub/clinvar/disease_names' \
+  | tail -n+2 \
+  | cut -f1 \
+  | sort -uf \
```

Comment:
Good point, I've replaced all occurrences of `sort -uf | shuf -n 100` with `sort -ufR | head -n 100`. It should only do one full pass of the records now instead of two